### PR TITLE
Build overhaul

### PR DESCRIPTION
**Pull request by Ancient123**

### CODE REVIEW

**Review by elliedavidson — APPROVED** [review]
LGTM!

File: .github/workflows/lints.yml [hunk 1]
```diff
@@ -4,33 +4,37 @@ on:
...
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y protobuf-compiler
+
+      - name: Install Just
+        run: |
+          wget https://github.com/casey/just/releases/download/1.14.0/just-1.14.0-x86_64-unknown-linux-musl.tar.gz
+          tar -vxf just-1.14.0-x86_64-unknown-linux-musl.tar.gz just
+          sudo cp just /usr/bin/just
```
Comment (DieracDelta) [inline .github/workflows/lints.yml:31]: Nit: could we do `apt-get install -y just` here? If so maybe we could just bundle it in with the previous step

Comment (Ancient123) [reply .github/workflows/lints.yml:31]: ( https://docs.makedeb.org/prebuilt-mpr/getting-started/ the apt repo info )

Comment (DieracDelta) [reply .github/workflows/lints.yml:31]: Ah, makes sense!

**Pull request closed without merging**